README.md

```diff
@@ -1,5 +1,5 @@
-![cocotb logo](/../../../../cocotb-web/blob/master/assets/img/cocotb-logo.svg#gh-light-mode-only)
-![cocotb logo](/../../../../cocotb-web/blob/master/assets/img/cocotb-logo-dark.svg#gh-dark-mode-only)
+![cocotb logo](/../../../../cocotb/cocotb-web/blob/master/assets/img/cocotb-logo.svg#gh-light-mode-only)
+![cocotb logo](/../../../../cocotb/cocotb-web/blob/master/assets/img/cocotb-logo-dark.svg#gh-dark-mode-only)
```

Comment:
```suggestion
![cocotb logo](https://www.cocotb.org/assets/img/cocotb-logo.svg#gh-light-mode-only)
![cocotb logo](https://www.cocotb.org/assets/img/cocotb-logo-dark.svg#gh-dark-mode-only)
```

Reply:
That doesn't work I don't think, I think they have to be relative for github to respect the hash

Reply:
Hm. And relative URLs won't work on PyPi or LibreCores where we use the same README. Maybe we have to put the logo into the cocotb source tree.

Reply:
Oh, good point. The whole light / dark mode thing won't work too well there anyway.

Reply:
#2915